### ISSUE TITLE
Joplin not syncing with Syncthing

### ISSUE THREAD

**Issue by coolioniki** [laurent22/joplin#6388]
I am using syncthing (most recent version) to sync my notes on Joplin (ver 2.5.5) between my laptop (xps 9510) and phone (samsung s21).  I am using a previous version of Joplin because the current version of Joplin from the playstore does not work at all with syncthing. I am able to sync my files from Joplin to syncthing on my laptop, and then to syncthing on my phone. However, I am unable to sync my files on syncthing on my phone to Joplin on my phone. It just says it's syncing with the '[sync icon] cancel' and doesn't sync (I have a single notebook with a singke file). If I cancel the sync, it says on 'Cancelling...' and doesn't change.

tl:dr - joplin mobile is stuck on syncing 

## Environment

Joplin version: 2.5.5
Platform: windows 10, android 12

## Steps to reproduce

1. create notebook and note on laptop, sync with syncthing
2. click sync on joplin app 
3. wait

## Describe what you expected to happen
notebooks and notes would sync, so I can see the notes from my computer Joplin on my phone Joplin

## Logfile
I have an 'interactive bug report' but I don't know what kind of information (device/person) is on it sorry.

**Comment by coolioniki** [comment]
Think i fixed it!
Sync folder on phone should be
/storage/emulated/0/joplin
not
storage/emulated/0/joplin (missing forward dash)
